src/jquery.uls.core.js

```diff
@@ -115,6 +115,11 @@
 		 * @return {Object}
 		 */
 		position: function () {
+			if ( this.options.onPosition ) {
```

Comment:
this has to unfortunately go under the "var" declarations

Reply:
Done.